qa/rolling-upgrade/src/test/java/org/elasticsearch/upgrades/IndexingIT.java

```diff
@@ -171,6 +166,62 @@ public void testAutoIdWithOpTypeCreate() throws IOException {
         }
     }
 
+    public void testDateNanosFormatUpgrade() throws IOException {
+        final String indexName = "test_date_nanos";
+        switch (CLUSTER_TYPE) {
```

Comment:
I think we can use `UPGRADE_FROM_VERSION` instead of `minNodeVersion` as we do not execute anything in a mixed cluster.

Reply:
UPGRADE_FROM_VERSION is available in the x-pack upgrade tests but not in qa.

Reply:
++, I use it in the [backport](https://github.com/elastic/elasticsearch/pull/70904) to disable the test with 6.8 (date_nanos was added in 7.0).